recipes/Open3D/meta.yaml

```diff
@@ -0,0 +1,120 @@
+{% set name = "Open3D" %}
+{% set version = "0.18.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/isl-org/Open3D/archive/v{{ version }}.tar.gz
+  sha256: 524ddeb7dc8aaed6dc5b272415df0c8ffcf7eff0816a84cfbdee56cdd6d3587a
+  patches:
+    - fix-tbb.patch
+    - fix-unzip.patch
+    - fix-curl.patch
+    - fix-error.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - pkg-config
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+    - {{ cdt('mesa-libgl-devel') }}          # [linux]
+    - {{ cdt('mesa-dri-drivers') }}          # [linux]
+    - {{ cdt('mesa-libegl-devel') }}         # [linux]
+    - {{ cdt('libselinux') }}                # [linux]
+    - {{ cdt('libxdamage') }}                # [linux]
+    - {{ cdt('libxfixes') }}                 # [linux]
+    - {{ cdt('libxxf86vm') }}                # [linux]
+    - {{ cdt('libxcb') }}                    # [linux]
+    - {{ cdt('libxext') }}                   # [linux]
+    - {{ cdt('xorg-x11-server-xvfb') }}      # [linux]
+    - {{ cdt('libxau') }}                    # [linux]
+    - {{ cdt('libxi-devel') }}               # [linux]
+    - python                                 # [build_platform != target_platform]
+    - pip                                    # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+  host:
+    - setuptools
+    - python
+    - pip
+    - libboost-devel
+    - assimp
+    - eigen
+    - glew
+    - glfw
+    # fmt 10 results in build error
+    - fmt 9.*
+    - libjpeg-turbo
+    - gtest
+    - gmock
+    - jsoncpp
+    - zeromq
+    - cppzmq
+    - msgpack-c
+    - msgpack-cxx
+    - minizip
+    - vtk
+    - libidn2
+    - libpng
+    - pybind11
+    - qhull
+    - tinyobjloader
+    - libblas
+    - libcblas
+    - liblapack
+    - liblapacke
+    # - nanoflann  # TODO: Does not export cmake config in conda-forge; https://github.com/conda-forge/nanoflann-feedstock/issues/4
+    # - tinygltf   # TODO: Does not export cmake config in conda-forge
+    # - libcurl    # does not like the conda-forge version, results in linker error
+    # - openssl    # does not like the conda-forge version, results in linker error
+    # - tbb-devel  # conda-forge version too new
+    # - imgui      # Not on conda-forge
+    # - libzlf     # Not on conda-forge
```

Comment:
Please add additional recipes to this PR for imgui and zlf. They both have multiple stable releases to choose from and already have USE_SYSTEM_* options implemented in the CMakeLists for open3d.

Reply:
Have added.